src/main/scala/is/hail/expr/FunctionRegistry.scala

```diff
@@ -2774,6 +2774,150 @@ object FunctionRegistry {
   register("//", (x: Float, y: Float) => math.floor(x / y).toFloat, null)
   register("//", (x: Double, y: Double) => math.floor(x / y), null)
 
+  register("//", { (xs: IndexedSeq[Int], y: Int) =>
```

Comment:
@danking is there a better way to do this stuff? I didn't see one that didn't involve implicit numeric stuff, which I think we should avoid.

Reply:
wait, crap. I think I have to handle missingness in here.

Reply:
I think you're looking for `registerNumeric`, why do you want to avoid it?